modules/codegen/src/main/scala/com/twilio/guardrail/ProtocolGenerator.scala

```diff
@@ -95,37 +182,41 @@ object ProtocolGenerator {
       * so essentially we have to return false if:
       *   - there are any uppercase characters
       */
-    def couldBeSnakeCase(s: String): Boolean = s.toLowerCase(Locale.US) == s
```

Comment:
This function was moved without its comment

Reply:
I really didn't want it. Couldn't we skip it?)

Reply:
Without moving the comment, now it just applies to nothing. Do you think there's a better place for this comment to live?

Reply:
Oops! I'm really sorry. I lost it.